pkg/parcacol/querier.go

```diff
@@ -1427,10 +1448,15 @@ func (q *Querier) selectMerge(
 	}
 
 	queryParts.Meta.Timestamp = start
+	queryParts.Meta.SampleType = resultType
```

Comment:
I'm not a fan of overriding a sub-struct of queryParts like this, we don't use anything else of queryParts

Reply:
We return queryParts because those are now passed into SymbolizeArrowRecord to check for delta profiles.